cmake_modules/VCMI_lib.cmake

```diff
@@ -0,0 +1,530 @@
+macro(add_main_lib TARGET_NAME LIBRARY_TYPE)
```

Comment:
Is it now a correct place to add files from VCMI_lib  to?

Reply:
yes, exactly